Answers.md

```diff
@@ -1,2 +1,28 @@
 # Your responses to the short answer questions should be laid out here using Mark Down.
 ### For help with markdown syntax [Go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+1. .forEach merely iterates through an array while .map actually iterates and takes the elements to make a new array.
```

Comment:
A clear difference is that forEach doesn't return anything whereas map does! Good answer!